twitter.css

```diff
@@ -0,0 +1,316 @@
+    margin-left: 10px;
+    align-items: center;
+}
+.home-logo{
+    width: 35px;
+    height: 35px;
+}
+
+.tweet{
+    background-color: #1DA1F2;
+    border-radius: 50px;
+    height: 50px;    
+    width: 250px;
+    margin-top: 13px;
+}
+.tweet > *{
+    margin:0px;
+    justify-content: center;
+    text-align: center;
+    padding: 12px;
+    color: white;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+#small-profile{
+    height: 35px;
+    width: 35px;
+}
+.profile-name > *{
+    margin: 0px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+a{
+    text-decoration: none;
+    color: black;
+}
+a:hover{
+    color: #1DA1F2;
+}
+
+
+
+@media screen and (max-width: 1580px) {
+    .main-body {
+            display: grid;
+            grid-template-columns: 70px 4fr 1.8fr;
+            background-color: var(--first-color-light);
+        }
+    .icons h2 {
+            display: none;
+        }
+    .tweet{
+        display: none;
+    }
+    .profile p ,
+    .profile .dots{
+        display: none;
+    }
+
+    }
+    .icons{
+        margin-bottom: 20px;
+        margin-top: 20px;
+    }
+    
+    
+
+/* Main Section */
+.main{
+    border: 1.5px solid #E1E8ED;
+}
+
+.sparkles{
+    height: 50px ;
+    width: 50px;
+}
+
+.home-main{
+    align-items: center;
+
+}
+.home{
+    margin: 15px;
+}
+
+.profile-icon{
+    height: 50px;
+    width: 50px;
+    padding: 5px;
+    margin-left: 5px;
+}
+.status-bar{
+    border-top: 1.5px solid #E1E8ED;
+    border-bottom: 1.5px solid #E1E8ED;
+    
+}
+.upper-bar{
+    align-items: center;
+}
+.happening{
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    font-size: 23px;
+    color: grey;
+    font-weight: 100;
+}
+.status-icons > *{
+    height: 25px;
+    width: 25px;
+    margin-left: 15px;
+}
+.lower-bar{
+    align-items: center;
+    column-gap: 145px;
+    margin-left: 58px;
+}
+.tweet-status > *{
+    font-size: 18px;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+.tweet-status{
+    background-color: #1DA1F2;
+    border-radius: 50px;
+}
+.tweet-status h3{
+    display: grid;
+    place-content: center;
+    color: white;
+    height: 12px;
+    
+}
+
+.statement-icons > *{
+    height: 15px;
+    width: 15px;
+}
+
+.statements{
+    height: auto;
+    width: auto
+    ;
+}
+.boxes {
+    height: auto;
+    border-radius: 25px;
+}
+    
+.user{
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+     font-size: small;
+    color: #AAB8C2;
+}
+.profile-statement p {
+    color: black;
+font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;    
+font-weight: 500;
+
+}
+
+
+.paragraph{
+    font-family: 'Franklin Gothic Medium',
+     'Arial Narrow', Arial, sans-serif;
+    font-size: 14px;}
+
+.right-icon{
+    height: 30px;
+    width: 30px;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+/* News Right side Bar */
```

Comment:
Well done for commenting your sections. It helps you to remember the sections when you come back to this project, and it helps other people reading your code to understand the sections.